### ISSUE TITLE
Parametr `rp.enable=false` causes java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by dkozlov** [reportportal/agent-java-cucumber#5]
```
java.lang.NoSuchMethodError: com.epam.reportportal.service.ReportPortal.startLaunch(Lcom/epam/reportportal/service/ReportPortalClient;Lcom/epam/reportportal/listeners/ListenerParameters;Lcom/epam/ta/reportportal/ws/model/launch/StartLaunchRQ;)Lcom/epam/reportportal/service/ReportPortal;
	at com.epam.reportportal.cucumber.AbstractReporter$1.get(AbstractReporter.java:89)
	at com.epam.reportportal.cucumber.AbstractReporter$1.get(AbstractReporter.java:71)
	at rp.com.google.common.base.Suppliers$MemoizingSupplier.get(Suppliers.java:120)
	at com.epam.reportportal.cucumber.ScenarioReporter$1.get(ScenarioReporter.java:62)
	at com.epam.reportportal.cucumber.ScenarioReporter$1.get(ScenarioReporter.java:55)
	at rp.com.google.common.base.Suppliers$MemoizingSupplier.get(Suppliers.java:120)
	at com.epam.reportportal.cucumber.ScenarioReporter.getRootItemId(ScenarioReporter.java:106)
	at com.epam.reportportal.cucumber.AbstractReporter.beforeFeature(AbstractReporter.java:125)
	at com.epam.reportportal.cucumber.AbstractReporter.feature(AbstractReporter.java:297)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cucumber.runtime.Utils$1.call(Utils.java:40)
	at cucumber.runtime.Timeout.timeout(Timeout.java:16)
	at cucumber.runtime.Utils.invoke(Utils.java:34)
	at cucumber.runtime.RuntimeOptions$1.invoke(RuntimeOptions.java:294)
	at com.sun.proxy.$Proxy38.feature(Unknown Source)
	at cucumber.runtime.junit.JUnitReporter.feature(JUnitReporter.java:184)
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:69)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:95)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:38)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.api.junit.Cucumber.run(Cucumber.java:100)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:283)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:173)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:128)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
```

**Comment by avarabyeu** [comment]
@dkozlov agent version?

**Comment by dkozlov** [comment]
3.0.0

**Comment by avarabyeu** [comment]
Fixed in 3.0.1

**Comment by Mikalai-Kabzar** [comment]
Hi, guys.
Please update maven dependency on 
https://github.com/reportportal/agent-java-cucumber page.
 "<version>3.0.0-BETA-1</version>" contains this bug.
Please change to "<version>3.0.1</version>"
3.0.1 works fine.
